index.bs

```diff
@@ -1111,7 +1113,7 @@ The modules spec should only invoke this operation from methods of Source Text M
 1. Assert: _entry_ must have all of the internal slots of a ModuleStatus Instance (<a href="#module-status-internal-slots">5.5</a>).
 1. Assert: _result_ must be a List.
 1. If _entry_ is already in _result_, return *undefined*.
-1. Append _entry_ to _result_.
+1. Insert _entry_ as the first element of _result_.
```

Comment:
small detail that might be completely irrelevant but it means the deps collection is now in evaluation order.